src/table/table.stories.ts

```diff
@@ -97,10 +162,15 @@ class TableStory implements OnInit, OnChanges {
 			[striped]="striped"
 			[isDataGrid]="isDataGrid"
 			(sort)="customSort($event)">
+			<ibm-table-toolbar [model]="model">
```

Comment:
I think the toolbar needs to move outside of the table ... I'm not seeing it rendered in the story 🤔

Reply:
Hmm, that's weird... it's working for me 🤔